src/plugins/import-notifications/server/controllers/parseNotification.ts

```diff
@@ -0,0 +1,35 @@
+export interface NotificationRaw {
+  templateId: number
+  data: any
+  account: string
+}
+
+export function parseNotification(rows: string[][]): NotificationRaw[] {
+  return rows.slice(1).reduce<NotificationRaw[]>((acc, row, index) => {
+    if (typeof row[0] !== 'string') throw new Error('CSV file is invalid!')
+
+    const templateId = +row[0].trim()
+    const account = row[1].trim()
+    const data = parseData(row[2].trim())
```

Comment:
mmm, I thought the:
- data would be the 2nd, 3rd, 4th column, ... (while we have more columns)
- We would use the header of the CSV to name the fields for the template